next/docs/identity.rs/0.7-alpha/docs/libraries/wasm/getting_started.mdx

```diff
@@ -77,8 +77,9 @@ cat | sed -e "s#require('@iota/identity-wasm/node')#require('./node')#" | node
 
 ```typescript
```

Comment:
Would it make more sense to include this code as import? Does this example exist in your repo?

Reply:
Ideally, we would import it. But here in the README we would like to have something that is self-contained and can be copy-pasted and run immediately. If we import our `0_create_did.ts` example, it is not self-contained as it requires extra util functions for requesting funds from the faucet. Granted, this example here also needs a private tangle which is not exactly obvious to any user, so it can still be improved.